### PR TITLE
Kb behavior

### PR DESCRIPTION
**Pull request by keithbradley1**
created the initial tests in the behavior_predictor

### CODE REVIEW

**Comment by ashwoodaa** [comment]
:+1:

File: test_behavior_predictor.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+  def setUpClass(self):
+    ''' Our doc string '''
+    pass
+
+  def test_(self):
+    ''' Our doc string '''
+    pass
+
+  def test_for_receipt_of_value_for_dictator(self):
```
Comment (Jessicashinjo) [inline test_behavior_predictor.py:20]: Did you mean test_for_receipt_of_value_from_dictator_dictionary ?

**Comment by carter-harris** [comment]
👍

**Comment by Jessicashinjo** [comment]
👍

**Pull request merged**